Categories/UIView+TCAdditions.m

```diff
@@ -129,5 +129,19 @@ - (UIView *)tc_breadthFirstSubviewThatSatisfiesBlock:(BOOL (^)(UIView *))conditi
     return nil;
 }
 
+- (void)tc_autoLayoutPinToParentViewController {
```

Comment:
? - Should this be included? What is this bringing to the table that other AutoLayout libraries don't have? If we think that it could be better than current libraries, then I would suggest moving it into its own pod/library. Check out this one for reference: https://github.com/Raizlabs/RZUtils/blob/master/RZUtils/Categories/UIView/UIView%2BRZAutoLayoutHelpers.h